bfd/elfxx-riscv.c

```diff
@@ -1027,6 +1027,8 @@ riscv_elf_add_sub_reloc (bfd *abfd,
 }
 
 #define RISCV_UNKNOWN_VERSION -1
+#define RISCV_CRYPTO_MAIN_VERSION 9
+#define RISCV_CRYPTO_SUB_VERSION 0
```

Comment:
Please define the default version in the ext_version_table rather than here,
https://github.com/riscv/riscv-binutils-gdb/blob/riscv-binutils-experiment/gas/config/tc-riscv.c#L100